engine/movie/splash.asm

```diff
@@ -16,11 +16,11 @@ LoadShootingStarGraphics:
 	lb bc, BANK(FallingStar), (FallingStarEnd - FallingStar) / $10
 	call CopyVideoData
 	ld hl, GameFreakLogoOAMData
-	ld de, wOAMBuffer + $60
+	ld de, wShadowOAM + $60
```

Comment:
...you get the idea: use the labels generated by the `sprite_oam_struct` macro, and remove redundant comments.